lib/tasks/cuttlefish.rake

```diff
@@ -16,6 +16,18 @@ namespace :cuttlefish do
     Email.all.each {|email| email.update_status! }
   end
 
+  desc "Archive all emails created more than 6 months ago"
+  task :archive_6_month_old_emails => :environment do
+    date_of_oldest_email = Delivery.order(:created_at).first.created_at.to_date
+    if date_of_oldest_email < 6.months.ago.to_date
+      (date_of_oldest_email...6.months.ago.to_date).each do |date|
+        Archiving.archive(date)
```

Comment:
This method needs to be updated to fulfil [this requirement](https://github.com/mlandauer/cuttlefish/issues/232#issuecomment-280497491):

>This should also include removing the archive files from disk after they're transferred to S3, which is something we do manually at the moment.